.changeset/small-maps-give.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
```

Comment:
This should be a `patch` because it's a bugfix not new feature

Reply:
Changed to `patch` level, thank you!